### ISSUE TITLE
Exporting MANO hand and object meshes

### ISSUE THREAD

**Issue by wqdong8** [zdchan/GraspXL#18]
Hi, thank you for the excellent work and for releasing this project!

I have a question regarding exporting meshes.
**I would like to export both the MANO hand and the corresponding object as meshes**. Could you please advise on the simplest way to do this?

Currently, I am not fully clear about the meaning of trans, rot, and angle for the Objaverse objects(like data['object_id']['trans'] in .npy files). I am referencing [[this documentation]](https://github.com/zdchan/GraspXL/blob/main/docs/DATASET.md), but I’m still a bit confused. How should I use trans, rot, and angle to correctly move the object?

For the MANO hand, I modified the original [MANO repository code](https://github.com/otaheri/MANO) and was able to successfully export the hand mesh with the following code snippet:
```

import torch
import mano
from mano.utils import Mesh
import numpy as np

data = np.load("path/to/mano_dataset_1/large/000a8be2894d460d8a899577e2a569d1/mano_1.npy", allow_pickle=True)
model_path = 'models/mano'
n_comps = 45
batch_size = data.item()['right_hand']['rot'].shape[0]

rh_model = mano.load(model_path=model_path,
                     is_rhand= True,
                     num_pca_comps=n_comps,
                     batch_size=batch_size,
                     flat_hand_mean=False)

betas = torch.zeros(batch_size, 10)
pose = torch.rand(batch_size, n_comps)*.1
global_orient = torch.rand(batch_size, 3)
transl        = torch.rand(batch_size, 3)


rot = torch.from_numpy(data.item()['right_hand']['rot'])
pose = torch.from_numpy(data.item()['right_hand']['pose'])
trans = torch.from_numpy(data.item()['right_hand']['trans'])

output = rh_model(
    # betas=betas,                               
    global_orient=rot,   
    hand_pose=pose,      
    transl=trans,       
    return_verts=True,
    return_tips=True
)

h_meshes = rh_model.hand_meshes(output)


#visualize hand mesh only
h_meshes[0].show()

#visualize hand and joint meshes
hj_meshes = Mesh.concatenate_meshes([h_meshes[0]])
hj_meshes.show() 
```

Could you please clarify how I can also export the corresponding object mesh together with the hand?

Thanks a lot for your time and help!

**Comment by zdchan** [comment]
Hi,

As explained in [the document](https://github.com/zdchan/GraspXL/blob/main/docs/DATASET.md), 'trans' records the position sequences of the object, 'rot' records the axis-angle sequences of the object, and 'angle' is not used.
